test/test_linear_space_lcs.py

```diff
@@ -0,0 +1,60 @@
+import os
+import unittest
+
+from approximate_string_matching import linear_space_lcs, lcs
+from generator import rand
+
+LCS_ALGORITHMS = [
+    linear_space_lcs.linear_space_lcs,
+]
+
+class TestLinearSpaceLongestCommonSubsequence(unittest.TestCase):
+  run_large = unittest.skipUnless(
+      os.environ.get('LARGE', False), 'Skip test in small runs')
+
+  """Test if substr is subsequence of the word"""
+  def is_subsequence(self, word, substr):
+    j = 0
+    for i in range(0, len(word)):
```

Comment:
If you have "for i in range(len(x))", always replace it with "for i, c in enumerate(x)"

Reply:
Done